sync/pending.go

```diff
@@ -58,6 +58,70 @@ func (p *Pending) Variant() core.PendingDataVariant {
 	return core.PendingBlockVariant
 }
 
+func (p *Pending) GetPreLatest() *Pending {
+	return nil
+}
+
+type PendingDataWPreLatest interface {
```

Comment:
Nitpic: `W` of `with`? I think we can name it `PendingDataWithPreLatest` or maybe directly `PreLatestData`?